core/domain/services/graphql/connection_resolvers/FormSectionConnectionResolver.php

```diff
@@ -128,14 +128,13 @@ public function get_query_args(): array
      */
     public function sanitizeInputFields(array $where_args): array
     {
-        $arg_mapping = [
-            'appliesTo' => 'FSC_appliesTo',
-            'belongsTo' => 'FSC_belongsTo',
-            'status'    => 'FSC_status',
-        ];
         return $this->sanitizeWhereArgsForInputFields(
             $where_args,
-            $arg_mapping
+            [
+                'appliesTo' => 'FSC_appliesTo',
+                'belongsTo' => 'FSC_belongsTo',
+                'status'    => 'FSC_status',
+            ]
         );
```

Comment:
As per the suggestion [here](https://github.com/eventespresso/event-espresso-core/pull/3440#discussion_r658075179).